src/styles/stars.scss

```
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  padding: 0;
+  list-style: none;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--1 > :nth-child(-n+1),
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars__star,
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

apply styles to all but the last one

```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```